src/content/doc-surrealdb/security/troubleshooting.mdx

```diff
@@ -0,0 +1,96 @@
+---
+sidebar_position: 5
+sidebar_label: Troubleshooting
+title: Troubleshooting | Security
+description: This page provides some troubleshooting advice to support users in addressing issues either caused by or involved in the usage of specific security features provided by SurrealDB.
+---
+
+# Troubleshooting
+
+This page provides some troubleshooting advice to support users in addressing issues either caused by or involved in the usage of specific security features provided by SurrealDB.
+
+## Authentication
+
+### Invalid Authentication Error
+
+The most common error that users may receive when authenticating to SurrealDB is `InvalidAuth`, which results in a generic message such as `There was a problem with authentication`. The reason that this message is so vague and yet so common is that it is a placeholder for other more specific internal errors related to authentication. These errors are not returned to the client due to their potential to leak internal information about the database (e.g. whether or not a user or access method exists, whether a token failed to verify because of its signature or a specific claim...) to unauthenticated users.
+
+Although the internal cause for these errors is not revealed to clients, it can be identified by the SurrealDB administrator through its server logs. These logs will usually be prefixed by `surreal::iam::`. Most helpful messages will be displayed when starting SurrealDB with `--log debug`. If those logs are not enough to diagnose the problem, starting the server with `--log trace` will provide additional messages describing the authentication process.
```

Comment:
```suggestion
Although the internal cause for these errors is not revealed to clients, it can be identified by the SurrealDB administrator through its server logs. These logs will usually be prefixed by `surreal::iam::`. Most helpful messages will be displayed when [starting SurrealDB with `--log debug`](/docs/surrealdb/cli/start#command-help). If those logs are not enough to diagnose the problem, starting the server with `--log trace` will provide additional messages describing the authentication process.
```